http/src/main/scala/org/http4s/blaze/http/http2/FlowStrategy.scala

```diff
@@ -0,0 +1,43 @@
+package org.http4s.blaze.http.http2
+
+import org.http4s.blaze.http.http2.FlowStrategy.Increment
+
+/** The goal of the `FlowStrategy` is to advise when to do windows updates for inbound data */
+trait FlowStrategy {
+  /** Decide if the session window needs to send a WINDOW_UPDATE frame
+    *
+    * @note This must not mutate the [[SessionFlowControl]] in any way.
```

Comment:
Is this a standard scaladoc tag?  I'm not familiar with it.

Reply:
The @note? I believe so.